### DISCUSSION TITLE
Root in ramdisk

### DISCUSSION THREAD

**Discussion by rotragit** [ltsp/ltsp#211] [Q&A]
I would like to load all root partition in ramdisk so each client has it's own disk and have not to access NFS or NBD. Is that possible?
Thank you,
romano

**Comment by alkisg** [comment]
Sure, see IMAGE_TO_RAM in the [ltsp.conf manpage](https://ltsp.org/man/ltsp.conf/).

Of course you're expected to have a kiosk setup, otherwise you'd still need the server for /home.
